### DISCUSSION TITLE
Is this the new home of re?

### DISCUSSION THREAD

**Discussion by iMichka** [baresip/re#191] [Q&A] [answered]
Hi. Homebrew maintainer here (Homebrew is a package manager for macOS and Linux).

We were wondering what is the difference / link between this repo and https://github.com/creytiv/re?
The same holds for `rem`.

Is the `creytiv` repo not actively maintained, so you are now the "official" upstream? Or is this a fork that goes in a different direction (which means that a new release on https://github.com/creytiv/re might conflict with what you are shipping here).

For reference: https://github.com/Homebrew/homebrew-core/pull/91871/files#r774099932

**Comment by sreimers** [comment, marked as answer]
Hi, this fork is maintained for baresip and newer baresip versions (>= 1.0.0) are depend on it. Same for [baresip-rem](https://github.com/baresip/rem). We are preparing to migrate both to a new mono repo called [baresip-libs](https://github.com/baresip/baresip-libs), but that will take some time.

**Comment by iMichka** [comment]
Thanks. We still ship the package from the original repo, and would like to avoid to have to manage 2 packages with the same name, but different upstreams. In general there is only one "canonical" upstream repository.

Do you consider the original repo abandoned?

**Comment by sreimers** [comment]
I can't say much about the status of creytiv/re. I think it's not completely abandoned (see [Contributions](https://github.com/creytiv/re/wiki/Contributions)). Maybe you can create a new package like `baresip-re` and mark the `libre` package as conflict.

Or you compile the baresip package static:
```
Folder structure:
├── baresip
├── re
└── rem
```
```
make -C re libre.a
make -C rem librem.a
make -C baresip baresip STATIC=1
```
https://github.com/baresip/baresip/wiki/Install:-GIT-Version#static-build

The Makefile prefers the `../re/` build folder over a global install.

**Comment by iMichka** [comment]
What we would like to find out is if you want to make both these two libs available to everyone or it's just used for internal use. We must think long term here: if people start reporting bugs, are you providing support (even if limited)? Are these libraries going to evolve over time, with the new features / bigger changes?

**Comment by sreimers** [comment]
Both libraries (baresip/re and baresip/rem) could be used outside baresip for own projects and we support these use cases (like restund).
baresip/re already evolved with new features and bigger changes: https://github.com/baresip/re/releases. But we try to maintain ABI/API compatibility as much as possible.
